lib/mastodon/emoji_cli.rb

```diff
@@ -72,13 +73,61 @@ def import(path)
       say("Imported #{imported}, skipped #{skipped}, failed to import #{failed}", color(imported, skipped, failed))
     end
 
+    option :category
+    option :shortcode, type: :boolean
+    option :overwrite, type: :boolean
+    desc 'export PATH', 'Export emoji to a TAR GZIP archive at PATH'
+    long_desc <<-LONG_DESC
+      Exports custom emoji to 'export.tar.gz' at PATH.
+
+      The --category option dumps only the specified category. If the given category doesn't exist, all emoji will be exported.
+
+      The --overwrite option will overwrite an existing archive.
+    LONG_DESC
+
+    def export(path)
+      exported = 0
+      skipped = 0
+      failed = 0
+      category = options[:category] ? CustomEmojiCategory.find_by(name: options[:category]) : nil
+
+      export_file_name = File.join(path, 'export.tar.gz')
+      if options[:overwrite] || !File.file?(export_file_name)
```

Comment:
Use a guard clause instead of wrapping everything in another nesting level. If the file already exists and the overwrite option is *not* given, exit with code 1 and error message.

Reply:
👌